Pages/index.html

```diff
@@ -13,89 +13,82 @@
     <script src="https://cdn.jsdelivr.net/npm/swiper@11/swiper-bundle.min.js"></script>
 </head>
 <body>
-    <section class="sectionOne">
-        <h1>Cynthonia(Tone)</h1>
-        <hr>
-        <nav class="nav">
-          <ul class="navLinks">
-            <li>About</li>
-            <li>Education</li>
-            <li>Previous Projects</li>
-            <li>Works in Progress</li>
-          </ul>
-        </nav>
-        <hr>
+  <article class="page">
+    <section>
+      <h1>Cynthonia(Tone)</h1>
+      <hr>
+      <nav>
+        <ul>
+          <li>About</li>
+          <li>Education</li>
+          <li>Previous Projects</li>
+          <li>Works in Progress</li>
+        </ul>
+      </nav>
+      <hr>
     </section>
 
-    <section class="sectionTwo">
-        <div class="row sectionTwodiv">
+    <section>
+      <h2>Get To Know A Bit About Me</h2>
+        <div class="row secTwo">
           <img src="/Images/tanBlazer.jpeg" alt="Profile image of Cynthonia sitting at a table, in a tan blazer." class="profileImage">
-          <h2>Get To Know A Bit About Me</h2>
-          <div class="row secTwopara">
+          <div class="secTwopara">
+            <div class="secTwoScroll">
```

Comment:
the scrolling container was a little weird, so i added a second container inside of it.